cssom/Overview.bs

```diff
@@ -1457,10 +1462,35 @@ To <dfn>serialize a CSS rule</dfn>, perform one of the following in accordance w
 
 
  <dt>{{CSSMediaRule}}
- <dd>Issue: ...
+ <dd>
+  The result of concatenating the following:
+  <ol>
+   <li>The string "<code>@media</code>" followed by a single SPACE (U+0020).</li>
+   <li>The result of performing <a>serialize a media query list<a> on rule's media query list.</li>
+   <li>A single SPACE (U+0020), followed by the string "{", i.e., LEFT CURLY BRACKET (U+007B), followed by a single SPACE (U+0020).</li>
+   <li>The result of performing <a>serialize a CSS rule</a> on the each rule, separed by a single SPACE (U+0020), in rule's CSSRules list.</li>
+   <li>A single SPACE (U+0020), followed by the string "}", i.e., RIGHT CURLY BRACKET (U+007D)</li>
+  </ol>
 
  <dt>{{CSSFontFaceRule}}
- <dd>Issue: ...
+ <dd>
+  The result of concatenating the following:
+  <ol>
+   <li>The string "<code>@font-face {</code>" followed by a single SPACE (U+0020).</li>
```

Comment:
This matches Chromium and WebKit but Gecko and Edge use a newline. Gecko and Edge disagree on indentation though... So I guess this is OK.

http://software.hixie.ch/utilities/js/live-dom-viewer/saved/4683